blocks/masthead-block/features/masthead-block/masthead-block.scss

```diff
@@ -1,4 +1,10 @@
 .masthead-block--text {
   font-size: calculateRem(14px);
   line-height: calculateRem(16px);
+
+}
+
+a {
+  color: #191919;
```

Comment:
is this being changed to a global style of any `a` tag has that color and no text decoration? I think we'd want to scope this just to masthead block styles right?

Reply:
this should be fixed now